sass/px-context-browser-filter.scss

```diff
@@ -48,7 +49,7 @@ input.filter__box {
   height: 2rem;
   display: block;
   padding: 4px;
-  color: gray;
+  color: var(--px-input-text-color--placeholder, lightgray);
```

Comment:
This is the spec? It should always be the placeholder color, never the input text color, right?

Reply:
yep, spec says it should be the placeholder color